packages/vkui/src/components/PromoBanner/PromoBanner.tsx

```diff
@@ -53,6 +57,12 @@ export const PromoBanner = ({
   className,
   ...restProps
 }: PromoBannerProps) => {
+  if (process.env.NODE_ENV === 'development') {
+    warn(
+      'Используйте событие [VKWebAppShowBannerAd](https://dev.vk.com/mini-apps/monetization/ad/banners)',
+    );
```

Comment:
```suggestion
    warn(
      'Компонент устарел и будет удален в v6. Используйте событие VKWebAppShowBannerAd https://dev.vk.com/mini-apps/monetization/ad/banners',
    );
```

Консоль не парсит markdown-ссылки, к сожалению. Плюс стоит в уведомлении дать понять, до какого момента можно не обновляться. 😉

Reply:
Готово